stl/inc/coroutine

```diff
@@ -10,16 +10,13 @@
 #if _STL_COMPILER_PREPROCESSOR
 
 #ifdef _RESUMABLE_FUNCTIONS_SUPPORTED
-#define _STL4039_COROUTINE                                         \
-    "The contents of <coroutine> are not available with /await. "  \
-    "Remove /await or use /await:strict for standard coroutines. " \
-    "Use <experimental/coroutine> for legacy /await support."
-#pragma message(_WARNING_MESSAGE("STL4039", _STL4039_COROUTINE))
+_EMIT_STL_WARNING("STL4039", "The contents of <coroutine> are not available with /await. "
+                             "Remove /await or use /await:strict for standard coroutines. "
+                             "Use <experimental/coroutine> for legacy /await support.");
 #undef _STL4039_COROUTINE // TRANSITION, DevCom-1479701

```

Comment:
`_STL4039_COROUTINE` is no longer being defined, so it doesn't need to be undeffed. Ditto `_STL4038_COROUTINE` below.